pages/admin.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react'
+import AppNav from '../components/AppNav'
+import Footer from '../components/Footer'
+import useSWR from 'swr'
+
+import { useQuery } from '@apollo/react-hooks'
+import { GET_LESSONS } from '../graphql/queries'
+
+const SERVER_URL = process.env.SERVER_URL
+
+export const fetcher = (url: string) =>
+  fetch(url, { credentials: 'include' }).then(r => r.json())
+
+const AdminPage: any = () => {
```

Comment:
What is the link with this code and your PR [NEW COMPONENT] NavLink?

Reply:
oh shoot. lol i didn't realize this page code got pushed in as well. thanks for pointing that out